### DISCUSSION TITLE
Memory consumption in development

### DISCUSSION THREAD

**Discussion by danisal** [vercel/next.js#69412] [Help]
### Summary

The next-server is taking too much memory in development mode. Do you know if this is normal? It doesn't seem to me, and I'm unsure if it's because something is wrong on our side or if it's nextJS stuff.

<img width="726" alt="image" src="https://github.com/user-attachments/assets/77b378f9-f0d3-4f88-8343-c2a2f0baa076">

**Comment by Juneezee** [comment]
Many users have encountered the same issue, and there is an open issue tracking this https://github.com/vercel/next.js/issues/54708.

Unfortunately, I don't think there is anything we can do on our side apart from double-checking our code and make sure there is no obvious leaks.
